contract-examples/test/contract_deployer_allow_list.ts

```diff
@@ -42,11 +42,6 @@ describe("ExampleDeployerList", function () {
 
   });
 
-  it("should add contract deployer as owner", async function () {
-    const contractOwnerAddr: string = await contract.owner()
-    expect(owner.address).to.equal(contractOwnerAddr)
-  });
```

Comment:
Testing the OpenZeppelin modifier, nothing to do with precompiles or our code.

Reply:
Agree